modules/components/src/NestedTreeView/style.css

```diff
@@ -0,0 +1,44 @@
+/* the tree node's style */
+.tree-view {
+  overflow-y: hidden;
+}
+
+.tree-view_item {
+  /* immediate child of .tree-view, for styling convenience */
+}
+
+/* style for the children nodes container */
+.tree-view_children {
+  margin-left: 16px;
+}
+
+.tree-view_children-collapsed {
+  height: 0px;
+}
+
+.tree-view_arrow {
+  cursor: pointer;
+  margin-right: 6px;
+  display: inline-block;
+  -webkit-user-select: none;
```

Comment:
We should have an autoprefixer so you can leave the prefixes off in any css module.

Reply:
I guess this is also a good reason to bring the theme css files into the build process rather than having them hanging out as static assets... I'll add an issue for adding autoprefixers, for now I'll just take out the prefixes.